packages/react-server/src/forks/ReactFlightServerConfig.dom-legacy.js

```diff
@@ -9,9 +9,19 @@
 
 import type {Request} from 'react-server/src/ReactFlightServer';
 import type {ReactComponentInfo} from 'shared/ReactTypes';
+import type {ReactClientValue} from 'react-server/src/ReactFlightServer';
 
-export * from '../ReactFlightServerConfigBundlerCustom';
-export * from 'react-dom-bindings/src/server/ReactFlightServerConfigDOM';
+// Used to distinguish these contexts from ones used in other renderers.
+// E.g. this can be used to distinguish legacy renderers from this modern one.
+export const isPrimaryRenderer = true;
```

Comment:
Is this a modern renderer given it's file name ends with `.dom-legacy.js`? If it's not a modern renderer, don't we need to mark this a secondary renderer to be able to distinguish it from the modern ones?

I guess right now it doesn't really matter since `renderToMarkup` is `react-server` only where you can't read from React Context anyway.

Reply:
This was just inlined to leave it unchanged but I think this is actually unused now.

https://github.com/facebook/react/pull/30115